packages/nextjs/components/ExtensionCard.tsx

```diff
@@ -33,6 +33,8 @@ export const ExtensionCard = ({ extension, isCurated }: { extension: Extension;
     plausible("extensionCopyClick", { props: { id: githubRepo } });
   };
 
+  const installCommand = extension.installCommand || `npx create-eth@latest -e ${extension.name}`;
```

Comment:
nit: I think it's better to use `extension.branch` instead of `extension.name` since `name` has more chances to be different

Reply:
This should be the name. This should only be used with the curated extensions (the third-party extensions should have installCommand defined).